eventActions/bookmarkActions.js

```diff
@@ -11,14 +11,17 @@ class bookmarkActions {
 				.setColor('#0F9BF1')
 				.setTitle(`${swordsEmote} Knights of Academia Bookmark ${swordsEmote}`)
 				.setDescription('You asked to bookmark this post from the Knights of Academia server.')
-				.addField('Full Message', workingMessage)
 				.addField('From', workingMessage.author, true)
 				.addField('Link to Message', `[Jump to Message](${workingMessage.url})`, true)
 				.addField('Channel', workingMessage.channel);
 			const messageChunks = workingMessage.content.match(/[\s\S]{1,1024}/g);
 
 			for (const chunk of messageChunks) {
-				bookmarkEmbed.addField('Message', chunk);
+				bookmarkEmbed.addField('Full Message', chunk);
+				if(workingMessage.attachments.array().length > 0){
```

Comment:
Doing this in the `for` loop could result in the attachment link being added multiple times (as the max length is 2000 characters and the message is split for each 1024 characters, it could be added up to two times)

A better way could be to set the image of the embed to the URL of the attachment.

Reply:
To be honest, I'm not sure what led me to do that in the for loop so I'll definitely take that out.

In terms of the image, I went with the link in the event the attachment may not be an image but I also didn't know embeds had the set image field.

I'll go ahead and push an update to this PR in the start of the week that removes this from the for loop and checks if the attachment is an image. Thanks!

(I didn't mean for this to be a review, my bad)

Reply:
Went ahead and fixed that up... Found that `setImage` function and we should be all set! Let me know if you had something else in mind 
![image](https://user-images.githubusercontent.com/12663558/77851042-8f3df900-71a4-11ea-95fc-c3f0c67eae3a.png)